packages/gestalt/src/Switch.tsx

```diff
@@ -62,6 +70,16 @@ export default function Switch({ disabled = false, id, name, onChange, switched
     !disabled ? styles.sliderDark : styles.sliderLight,
   );
 
+  const sliderVrStyles = classnames(
+    {
+      [styles.blackSlider]: !switched,
```

Comment:
instead of black can we say enabledSlider?

Reply:
Done